<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FolderWizardSourcePage</class>
 <widget class="QWidget" name="FolderWizardSourcePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Please give this folder an alias:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="aliasLineEdit">
     <property name="placeholderText">
      <string>Music</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Which folder in your computer you want to backup/synchronize?</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="localFolderLineEdit">
       <property name="placeholderText">
        <string>/home/local1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="localFolderChooseBtn">
       <property name="text">
        <string>&amp;Choose...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>234</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
